File: app/src/main/java/com/franjo/android/bakingapp/ui/StepFragment.java
package com.franjo.android.bakingapp.ui;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.franjo.android.bakingapp.R;

/**
 * Created by Franjo on 3.11.2017..
 */

public class StepFragment extends Fragment{



    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {

        View rootView = inflater.inflate(R.layout.fragment_steps_item, container, false);


        return rootView;
    }
}

File: app/src/main/java/com/franjo/android/bakingapp/presentation/ui/steps/StepFragment.kt
package com.franjo.android.bakingapp.presentation.ui.steps

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.fragment.app.viewModels
import com.franjo.android.bakingapp.R
import com.franjo.android.bakingapp.databinding.FragmentStepBinding
import com.franjo.android.bakingapp.presentation.base.BaseFragment

class StepFragment : BaseFragment<FragmentStepBinding>() {

    override val fragmentView: Int = R.layout.fragment_step

    private val viewModel: StepViewModel by viewModels()

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_step, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        binding.viewModel = viewModel
    }



}